client/sites/marketing/connections/service-examples.jsx

```diff
@@ -27,7 +27,25 @@ import './service-examples.scss';
  * a method with the example's content.
  * @type {string[]}
  */
-export const SERVICES_WITH_EXAMPLES = [ 'google_plus', 'mastodon', 'bluesky' ];
+export const SERVICES_WITH_EXAMPLES = [
+	'bandpage',
```

Comment:
Have we got any services without examples? Perhaps we should flip this logic, although it will be going away soon.